README.md

```diff
@@ -74,9 +74,31 @@ Example of possible config.
                   }
          }
         ]},
-     {ergw_aaa_radius, [{server, {{127,0,0,1}, 1812, <<"secret">>}}]},
+     {ergw_aaa_radius, [
+        {server, {{127,0,0,1}, 1812, <<"secret">>}},
+        {termination_cause_mapping, [
+            {normal, 1},
+            {administrative, 6},
+            {link_broken, 2},
```

Comment:
Would it be possible with a description in README what this does? Who supplies ''link_broken'' and who gets ''2'' instead?

Reply:
Agree, but I not sure if we need add some description for now, because we still don't have a clear vision of default values. I suppose we will do it after latest updated of default values soon in next PR